virttest/env_process.py

```diff
@@ -1036,7 +1036,7 @@ def preprocess(test, params, env):
     # Write it as a keyval
     test.write_test_keyval(version_info)
 
-    libvirtd_inst = utils_libvirtd.Libvirtd()
+    libvirtd_inst = None
```

Comment:
Have to perform this in `postprocess` as well.
https://github.com/avocado-framework/avocado-vt/blob/a284e91d9f43b7289fffe94887d85ea794cb28ed/virttest/env_process.py#L1472-L1473

Reply:
@luckyh good catch.